### ISSUE TITLE
Question on FNM evaluation part

### ISSUE THREAD

**Issue by Nhatdealin** [mx54039q/FFGAN#27]
I am using your repo FNM for state-of-the-art in face normalization. I tried to do the evaluation of FNM (CMU Multi-PIE and IJB-A) on VGGFace and LightCNN again, but the result is not similar your paper. I have some questions on this:

- Which pre-trained LightCNN model you used? 4-layers version, 9-layers version,...?
- On VGGFace, do you need to pre-processing testing images before?
- You are using VGGFace or VGGFace2 for evaluation?

**Comment by mx54039q** [comment]
Sorry I can't remember some detail, as it has been long since this work.
1. Sorry I cant remember.
2. Yes. Face image is detected and aligned before testing.
3. I did not evaluate our method on VGGFace or VGGFace2. MPIE and IJB-A are used for evaluation.

**Comment by Nhatdealin** [comment]
@mx54039q 

I have a question on Loss and config of lambda to compute loss:

 Feature_loss is L_ip in paper and we will multiply with lambda_1 as in paper (value in lambda_1 in paper is 10) but in default of repository is 100. So which one is better to train in your result? 
![image](https://user-images.githubusercontent.com/32137330/174252008-a660939a-7622-4cf4-846b-dc4820e2acc8.png)
![image](https://user-images.githubusercontent.com/32137330/174251879-36bcc80e-54ed-4bb3-8673-ebd57b1a0a1b.png)
![image](https://user-images.githubusercontent.com/32137330/174253094-1e45341c-e016-436f-8e3d-bec5673f6969.png)

**Comment by mx54039q** [comment]
The lambda_1 is 100. I forgot to revise it in the paper. You can also modify it to adapt to your own dataset. :)

**Comment by tylister** [comment]
> I am using your repo FNM for state-of-the-art in face normalization. I tried to do the evaluation of FNM (CMU Multi-PIE and IJB-A) on VGGFace and LightCNN again, but the result is not similar your paper. I have some questions on this:
> 
> * Which pre-trained LightCNN model you used? 4-layers version, 9-layers version,...?
> * On VGGFace, do you need to pre-processing testing images before?
> * You are using VGGFace or VGGFace2 for evaluation?

Hi! Could you share the code of evaluation?
